src/Support/AliasArguments/ArrayKeyChange.php

```diff
@@ -39,7 +39,7 @@ private function changeKey(array $target, array $segments, string $replaceKey):
         $segment = array_shift($segments);
 
         if (empty($segments)) {
-            if (isset($target[$segment])) {
+            if (array_key_exists($segment, $target) && $replaceKey !== $segment) {
```

Comment:
I don't see this change fully covered.

When I test it with an `} else { throw new \RuntimeException('boom'); }`, that exception is never thrown.

Is it possible to cover this?

I checked also the other changes, they are 🤞

Reply:
The null_value field I added covers this it would never, alias if not using array_key_exists

Reply:
Makes sense, thanks 👍